build/gamemaster_entrypoint.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 set -x
+set -e
```

Comment:
Can sometimes be bad practice - read more [here](https://stackoverflow.com/q/19622198/4306044). Your call though

Reply:
If there's a line which I don't mind failing, I'll append `|| true` 

But since this is a Docker build which should be very predictable I'm cool with leaving set -e